zuul.d/pipelines.yaml

```diff
@@ -74,9 +72,19 @@
         label: 'post'
     trigger:
       github:
+        # Trigger if approval is given
+        - event: pull_request_review
+          action: submitted
+          state: approved
```

Comment:
will every approve cause `check-post` `re-run`?

Reply:
ah right, it will